src/code-examples/tabs/javascript.js

```diff
@@ -1,144 +1,143 @@
-      });
-      selectedTab.removeAttribute("tabIndex");
-    };
-    
-    const getActiveTab = () => {
-      let selectedTab;
-      tabsList.forEach((tab) => {
-        if (tab.getAttribute("aria-selected") === "true") selectedTab = tab;
-      });
-      return selectedTab;
-    };
-    
-    /* ====================== */
-    /* = HANDLE KEY PRESSES = */
-    /* ====================== */
-    
-    // Select tab to the right
-    const selectRightTab = () => {
-      let newTab;
-      if (tabsList.indexOf(getActiveTab()) + 1 === tabsList.length) {
-        newTab = tabsList[0];
-      } else {
-        newTab = tabsList[tabsList.indexOf(getActiveTab()) + 1];
-      }
-      const oldTab = tabsList[tabsList.indexOf(getActiveTab())];
-      oldTab.setAttribute("aria-selected", false);
-      let tabPanel = document.getElementById(
-        \`\${newTab.getAttribute("aria-controls")}\`\
-      );
-      resetActiveTabs();
-      resetTabIndex(newTab);
-      togglePanel(tabPanel);
-      newTab.setAttribute("aria-selected", true);
-    };
-    
-    // Select tab to the left
-    const selectLeftTab = () => {
-      let newTab;
-      if (tabsList.indexOf(getActiveTab()) - 1 < 0) {
-        newTab = tabsList[tabsList.length - 1];
-      } else {
-        newTab = tabsList[tabsList.indexOf(getActiveTab()) - 1];
-      }
-      const oldTab = tabsList[tabsList.indexOf(getActiveTab())];
-      oldTab.setAttribute("aria-selected", false);
-      let tabPanel = document.getElementById(
-        \`\${newTab.getAttribute("aria-controls")}\`\
-      );
-      resetActiveTabs();
-      resetTabIndex(newTab);
-      togglePanel(tabPanel);
-      newTab.setAttribute("aria-selected", true);
-    };
-    
-    // Select first tab
-    const selectFirstTab = () => {
-      const newTab = tabsList[0];
-      let tabPanel = document.getElementById(
-        \`\${newTab.getAttribute("aria-controls")}\`\
-      );
-      resetActiveTabs();
-      resetTabIndex(newTab);  
-      togglePanel(tabPanel);
-      newTab.setAttribute("aria-selected", true);
-    };
-    
-    // Select last tab
-    const selectLastTab = () => {
-      const newTab = tabsList[tabsList.length - 1];
-      let tabPanel = document.getElementById(
-        \`\${newTab.getAttribute("aria-controls")}\`\
-      );
-      resetActiveTabs();
-      resetTabIndex(newTab);
-      togglePanel(tabPanel);
-      newTab.setAttribute("aria-selected", true);
-    };
-    
-    // Detects key presses
-    document.addEventListener("keydown", (e) => {
-      switch (e.key) {
-        case "ArrowRight":
-          selectRightTab();
-          break;
-        case "ArrowLeft":
-          selectLeftTab();
-          break;
-        case "Home":
-          selectFirstTab();
-          break;
-        case "End":
-          selectLastTab();
-          break;
-        default:
-          break;
-      }
-    });
-  `;
+const javascript = `let tabsList = Array.from(document.querySelectorAll("[role=tab]"));
```

Comment:
again, we still want to add one layer of nesting inside the `const`

Reply:
Let me know what you think about the weird formatting. Here's the Sandpack with the extra tab in our files. 
<img width="265" alt="Screen Shot 2022-12-13 at 5 15 21 PM" src="https://user-images.githubusercontent.com/69602589/207456137-0dee722a-8fab-4fb2-8d40-7bcd3bfb49e5.png">